tests/manual/testselect.html

```diff
@@ -11,6 +11,7 @@
     <title>TestSelect</title>
   </head>
   <body>
+    <h1>Single select without group headers<h1>
```

Comment:
Should be `</h1>` to close in all cases, I think.

Reply:
Sure... too quickie :+1: